allauth/socialaccount/providers/oauth2/utils.py

```diff
@@ -17,9 +17,8 @@ def token_urlsafe(nbytes=None):
 
 
 def generate_code_challenge():
-    # minimum length of 43 characters, maximum length of 128 characters
-    nbytes = random.randint(43, 128)
-    code_verifier = token_urlsafe(nbytes)
+    # Create a code verifier with a length of 128 characters
+    code_verifier = token_urlsafe(96)
```

Comment:
**Issue**: `import random` has now become an unused import -- can you remove it?